bundles/org.openhab.binding.digiplex/src/main/resources/OH-INF/thing/area.xml

```diff
@@ -10,6 +10,7 @@
 		</supported-bridge-type-refs>
 		<label>Area</label>
 		<description>Area</description>
+		<semantic-equipment-tag>AlarmSystem</semantic-equipment-tag>
```

Comment:
Might be similar to other comments about introducing a new tag `AlarmZone`

Reply:
Yup.